pyaedt/circuit.py

```diff
@@ -1577,6 +1573,7 @@ def connect_circuit_models_from_multi_zone_cutout(
             Distance increment for adding models.
 
         Examples
+        --------
         command to get the input arguments.
```

Comment:
This should be a complete sentence that introduces the code example.

Reply:
@PipKat thanks I made some changes I hope it's clear. This method is actually not easy to explain.